### PR TITLE
Fix memory overload with rf611_weightedfits.C on Windows 32 bit

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: tutorials/roofit/rf611_weightedfits.C [hunk 1]
```diff
@@ -108,7 +108,11 @@ int rf611_weightedfits(int acceptancemodel=2) {
 
   //number of pseudoexperiments (toys) and number of events per pseudoexperiment
   constexpr unsigned int ntoys = 500;
+#if defined(_WIN32) && !defined(__x86_64__)
```
Comment (lmoneta) [inline tutorials/roofit/rf611_weightedfits.C:111]: I don't understand why changing the number of events for the dataset used for fitting each toy fixes the problem!

Comment (bellenot) [reply tutorials/roofit/rf611_weightedfits.C:111]: > I don't understand why changing the number of events for the dataset used for fitting each toy fixes the problem!

`data.add(RooArgSet(costheta, weight), weight.getVal());` makes the memory increase at each iteration and reach the limit for 32 bit

Comment (jalopezg-r00t) [reply tutorials/roofit/rf611_weightedfits.C:111]: IIRC, on Win32, the user-kernel split for the virtual address space is at 2GB, which means 2GB usable for the process.  This _might_ work, however, in Linux 32-bit as the user-kernel split is a bit higher (at 3GB).

Is it normal that the process is consuming up to 2GB of memory?  Might be that a memory leak somewhere, @lmoneta?

**Comment by bellenot** [comment]
Fixed by commit https://github.com/root-project/root/commit/b4060d7519b77bd481bbdb3d086c3d0076260909

**Pull request closed without merging**